<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="TipTiltWidget" name="tip_tilt_1" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>389</width>
      <height>290</height>
     </rect>
    </property>
   </widget>
   <widget class="TipTiltWidget" name="tip_tilt_2" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>389</width>
      <height>290</height>
     </rect>
    </property>
   </widget>
   <widget class="TipTiltWidget" name="tip_tilt_3" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>389</width>
      <height>290</height>
     </rect>
    </property>
   </widget>
   <widget class="TipTiltWidget" name="tip_tilt_4" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>389</width>
      <height>290</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1422</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TipTiltWidget</class>
   <extends>QWidget</extends>
   <header location="global">tiptiltwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
